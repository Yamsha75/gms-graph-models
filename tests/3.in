4 91
4 242
4 161
4 441
4 184
4 369
4 444
4 507
4 259
4 160
4 199
4 308
4 466
4 423
4 206
4 361
4 54
4 453
4 485
4 382
4 422
4 504
4 418
4 113
4 17
4 186
4 216
4 275
4 106
4 81
4 294
4 475
4 164
4 173
4 79
4 457
4 33
4 439
4 248
4 139
4 114
4 78
4 19
4 430
4 154
4 156
4 359
4 213
4 250
4 25
4 185
4 431
4 497
4 228
4 389
4 119
4 60
4 420
4 41
4 4
4 155
4 196
4 366
4 346
4 165
4 375
4 210
4 438
4 376
4 276
4 182
4 45
4 311
4 224
4 463
4 265
4 212
4 284
4 77
4 219
4 374
4 234
4 324
4 23
4 351
4 349
4 257
4 318
4 92
4 258
4 255
4 211
4 18
4 235
4 465
4 436
4 487
4 271
4 127
4 325
