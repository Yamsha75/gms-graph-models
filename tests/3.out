7764
56875
24880
191014
32631
133342
193642
252988
65252
24567
38256
92594
213464
175624
41035
127574
2649
201634
231362
142988
174788
249982
171464
12098
230
33355
45175
73652
10614
6114
84292
221852
25831
28792
5808
205238
948
189272
59767
18456
12318
5658
292
181532
22731
23335
126152
43912
60747
526
32992
182384
243038
50407
148322
13448
3297
173122
1492
8
23032
37095
131164
117104
26152
137752
42667
188404
138494
74194
31915
1812
94424
48631
210704
68342
43495
78602
5510
46456
137012
53131
102562
440
120544
119162
64238
98764
7940
64744
63232
43080
260
53592
212542
186674
233288
71504
15360
103202
